### ISSUE TITLE
valid-describe-callback reports false positive with named function

### ISSUE THREAD

**Issue by msorens** [playwright-community/eslint-plugin-playwright#339]
I have a function that runs a block of tests and I use this function in a couple places, so I have it encapsulated as a first-class function rather than inlined, as is typical. The function is a valid callback meeting the criteria specified by the rule:

- should not be async
- should not contain any parameters
- should not contain any return statements

Minimal test case:
```
function foo() {
  test('create policies', async ({ system, page }) => {})
}
test.describe('policies', foo)
```
The `describe` line is flagged as a violation because the function is not inlined. Would be nice if the plugin could recognize this case.

**Comment by mskelton** [comment]
Why would you ever need to write tests like this?

**Comment by msorens** [comment]
Please re-read my first sentence; it is something I use in more than one place.

**Comment by mskelton** [comment]
I did read it.  There is just not scenario where that would make sense. It is not at all wrong for the plugin to complain about code that is poorly written.  That's a hill I will die on.  There are tons of scenarios that do not make sense that are "technically" valid, it does not mean I'm going to support them in the plugin.

Why aren't you using parameterized tests if you need to use a common test function? https://playwright.dev/docs/test-parameterize
